spec/requests/workflows/create_workflow_spec.rb

```diff
@@ -24,11 +24,23 @@
       allow(SendUpdateMessage).to receive(:publish)
     end
 
-    context 'when the version exists' do
+    context 'when the version is passed' do
       it 'creates new workflows' do
         headers = { 'CONTENT_TYPE' => 'application/xml' }
         expect do
-          put "/#{repository}/objects/#{druid}/workflows/#{workflow}", params: request_data, headers: headers
+          put "/#{repository}/objects/#{druid}/workflows/#{workflow}?version=1", params: request_data, headers: headers
+        end.to change(WorkflowStep, :count).by(16)
+
+        expect(SendUpdateMessage).to have_received(:publish).with(druid: druid)
+      end
+    end
+
+    context 'when the version exists in dor-services-app (deprecated)' do
```

Comment:
add version is not passed, please.

Reply:
isn't line 27 taking care of that?

Reply:
in specs before this one, there is "when the version is passed" clause and then right after "when the version is not passed" clause, which makes it clear what is diff.